### ISSUE TITLE
Does it work on Hypixel?

### ISSUE THREAD

**Issue by xMdb** [PrismarineJS/prismarine-web-client#251]
Has anyone tried to use it on Hypixel? Did you get banned? And does this web client send any weird packets to the server that Watchdog could potentially identify as a modified client?

**Comment by AwesomestCode** [comment]
Prismarine Web Client does not currently support online mode servers. In the future, please direct such queries to the discord server. Thanks!

**Comment by rom1504** [comment]
The issue is valid.
The answer is it probably works with some minor tweaks to auth

**Comment by xMdb** [comment]
> Prismarine Web Client does not currently support online mode servers. In the future, please direct such queries to the discord server. Thanks!

oh didn't know that.



> The issue is valid. The answer is it probably works with some minor tweaks to auth

Yes it probably would. I'm just wondering as I'd like to make a project using this to view Hypixel replays through a web browser. That would require logging onto Hypixel, but depending on what packets are (and aren't) being sent, Watchdog will probably ban the account.

**Comment by rom1504** [comment]
lot of people use mineflayer to connect to hypixel so I bet something like that could be made to work

**Comment by xMdb** [comment]
Yes that’s right, I also use Mineflayer 24/7 on Hypixel. I’m just wondering
if it will send any weird packets
-- 
From Matt

**Comment by U5B** [comment]
> Yes that’s right, I also use Mineflayer 24/7 on Hypixel. I’m just wondering if it will send any weird packets -- From Matt

Currently Mineflayer does not send a response to transaction packets (which hypixel spams in lobbies)
https://g.usbwire.net/i/🔑usb-219..png
A PR is open to fix that https://github.com/PrismarineJS/mineflayer/pull/2217

**Comment by xMdb** [comment]
> > Yes that’s right, I also use Mineflayer 24/7 on Hypixel. I’m just wondering if it will send any weird packets -- From Matt
> 
> Currently Mineflayer does not send a response to transaction packets (which hypixel spams in lobbies) [https://g.usbwire.net/i/🔑usb-219..png](https://g.usbwire.net/i/%F0%9F%94%91usb-219..png) A PR is open to fix that [PrismarineJS/mineflayer#2217](https://github.com/PrismarineJS/mineflayer/pull/2217)
> 
> Since your bot goes immediately to Limbo, I assume it is fine as there are no anticheat checks in Limbo (or Housing).

Didn't know that. Thanks! I'm looking to make a bot that views Hypixel replays on a web browser, using prismarine web client, would any movement packets be an issue or is it vanilla?

**Comment by U5B** [comment]
> > > Yes that’s right, I also use Mineflayer 24/7 on Hypixel. I’m just wondering if it will send any weird packets -- From Matt
> > 
> > 
> > Currently Mineflayer does not send a response to transaction packets (which hypixel spams in lobbies) [https://g.usbwire.net/i/🔑usb-219..png](https://g.usbwire.net/i/%F0%9F%94%91usb-219..png) A PR is open to fix that [PrismarineJS/mineflayer#2217](https://github.com/PrismarineJS/mineflayer/pull/2217)
> > Since your bot goes immediately to Limbo, I assume it is fine as there are no anticheat checks in Limbo (or Housing).
> 
> Didn't know that. Thanks! I'm looking to make a bot that views Hypixel replays on a web browser, using prismarine web client, would any movement packets be an issue or is it vanilla?

I don't know. Prismarine-web is not vanilla so there could be some unknown issues. But no one has tried yet.

**Comment by U5B** [comment]
> Didn't know that. Thanks! I'm looking to make a bot that views Hypixel replays on a web browser, using prismarine web client, would any movement packets be an issue or is it vanilla?

Correction: speed and slowness is currently messed up a tiny bit, so if Hypixel gives you speed, Mineflayer will go too fast.
https://github.com/PrismarineJS/prismarine-physics/pull/51
This PR should fix that once it is merged into prismarine-physics.

**Comment by xMdb** [comment]
> > Didn't know that. Thanks! I'm looking to make a bot that views Hypixel replays on a web browser, using prismarine web client, would any movement packets be an issue or is it vanilla?
> 
> Correction: speed and slowness is currently messed up a tiny bit, so if Hypixel gives you speed, Mineflayer will go too fast. [PrismarineJS/prismarine-physics#51](https://github.com/PrismarineJS/prismarine-physics/pull/51) This PR should fix that once it is merged into prismarine-physics.

Legend
